src/index.html

```diff
@@ -14,8 +14,46 @@
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img 
+        class="card__img" 
+        src="images/imac.jpeg" 
+        alt="iMac"
+      >
+
+      <p class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__code">Product code: 195434</p>
```

Comment:
```suggestion
      <h3 class="card__description">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h3>
      
      <p class="card__code">Product code: 195434</p>
```
It can be a h3\h4 tag, as it's a title of the card